src/tests/unit/tests/background/global-action-creators/user-configuration-action-creator.test.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { isFunction } from 'lodash';
+import { IMock, It, Mock, Times } from 'typemoq';
+
+import {
+    SaveIssueFilingSettingsPayload,
+    SetHighContrastModePayload,
+    SetIssueFilingServicePayload,
+    SetIssueFilingServicePropertyPayload,
+    SetIssueTrackerPathPayload,
+    SetTelemetryStatePayload,
+} from '../../../../../background/actions/action-payloads';
+import { UserConfigurationActions } from '../../../../../background/actions/user-configuration-actions';
+import { UserConfigurationActionCreator } from '../../../../../background/global-action-creators/user-configuration-action-creator';
+import { Interpreter } from '../../../../../background/interpreter';
+import { Action } from '../../../../../common/flux/action';
+import { Messages } from '../../../../../common/messages';
+import { TelemetryEventSource, TriggeredBy } from '../../../../../common/telemetry-events';
+
+describe('UserConfigurationActionCreator', () => {
+    it('handles GetCurrentState message', () => {
+        const payload = null;
+
```

Comment:
Consider grouping the code into Arrange/Act/Assert chunks rather than double-spacing each line?